gapic/src/main/com/google/gapid/perfetto/models/BatterySummaryTrack.java

```diff
@@ -107,6 +108,14 @@ protected Values createValues(int numRows, BinaryOperator<Values> combiner) {
     return new Values(numRows, combiner);
   }
 
+  private static CounterInfo trimPowerRailTrackName(CounterInfo powerRailTrack) {
```

Comment:
Maybe move this to just under the `onlyOne` function since this is more similar in functionality

Reply:
Makes sense. Updated